binscripts/rvm-installer

```diff
@@ -783,27 +783,33 @@ follow this link for details how to fix:
 
 rvm_install_validate_volume_mount_mode()
 {
-  \typeset path partition
+  \typeset path partition test_exec
 
   path=$rvm_path
 
-  # $rvm_path might not exists at this point so we need to traverse the directory tree upwards
-  # to find the name of the partition and then check if it has not been mount with noexec mode
+  # Directory $rvm_path might not exists at this point so we need to traverse the tree upwards
   while [[ -n "$path" ]]
```

Comment:
I am wondering if we should traverse the tree or rather create the folder? @mpapis what you think?